src/profiling.cpp

```
@@ -1,247 +1,247 @@
+                agg1, agg2, agg1*1.0/agg2);
+    }
+
+    double res, max_ = 0, min_=1e10;
+    for (int i=0; i<nthreads; i++) {
+        double val = (tprof[ALIGN1][i]*1.0) / tprof[MEM_CHAIN][i];
+        res += val;
+        if (max_ < val) max_ = val;
+        if (min_ > val) min_ = val;
+    }
+    fprintf(stderr, "\tAvg. FM-index traversal per get_sa_entry(): avg: %lf, max: %lf, min: %lf\n",
+            res/nthreads, max_, min_);
```

Cppcheck (2.20.0)

[warning] Uninitialized variable

(uninitvar)